.ci/kitchen-centos7-py2-cloud

```diff
@@ -0,0 +1,18 @@
+@Library('salt@master-1.5') _
+
+runTestSuite(
+    concurrent_builds: 0,
+    distro_name: 'centos',
+    distro_version: '7',
+    env: env,
+    golden_images_branch: 'master',
+    jenkins_slave_label: 'kitchen-slave',
+    kitchen_platforms_file: '/var/jenkins/workspace/nox-cloud-platforms.yml',
+    nox_env_name: 'runtests-zeromq',
```

Comment:
`runtests-cloud`

Reply:
I'll have to fix that on master too then, must have missed that earlier.

Reply:
If we merge the macos on nox PR, it fixes it.